modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -47,12 +39,15 @@ $ oc patch MachineConfigPool worker --type='merge' --patch \
   '{ "spec":{ "paused" :true } }'
 ----
 
-. To configure the OpenShift SDN cluster network provider, enter the following command:
+. To start the migration, set cluster network provider back to OpenShiftSDN by entering the following commands:
```

Comment:
s/set cluster/set the cluster/

Similar to above, is there a reason that "OpenShiftSDN" is not spaced here as it is elsewhere? If not, I'd make it "OpenShift SDN". If it's referring to a literal value, should it be in backticks?